interface/main/tabs/templates/patient_data_template.php

```diff
@@ -69,7 +69,7 @@ class="img-thumbnail"
                     </a>
                 <!-- /ko -->
                 <!-- ko if:!selectedEncounter() -->
-                    <?php echo xlt("None{{Encounter}}") ?>
+                    <?php echo xlt("None"). xlt("{{Encounter}}") ?>
```

Comment:
These are part of the translation constant. They are comments letting translating know a context if needed. It will translate if there is a None{{Encounter}} that has been translated. That reminds me; it's getting near time to do another translation run to get all these new constants into the translation spreadsheet.

Reply:
hi @bradymiller so if I understand you correctly I need to add the "None{{Encounter}}" constant to the translations ?

Reply:
hi @eyalvo-matrix ,
Yep, you'll need to add it to your other 10000 plus custom translations that your group already has :)
(I'll likely do a run to get this into the official spreadsheet and tables over the next couple weeks)
-brady

Reply:
Thanks , will do just that :) 
So this branch could be removed :+1: